CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -50,21 +51,21 @@ def get_dict_form(self) -> dict:
             "messsage": self.message,
             "success": self.success,
             "status_code": self.status_code,
-            "db_item": self.db_items,
+            "json_data": self.json_data,
         }
 
     def api_format(self) -> dict:
+        if self.json_data is not None:
```

Comment:
nit: `if self.json_data:`

Reply:
did some research and it looks like `if self.json_data` is faster than `self.json_data is not None`. 

And I don't think there will be cases where json_data is falsy other than None so I'll make the change. Thank you :)

The only time might be when a query is done and 0 results are found. json_data will be "[]". But that's not falsy so we're good

Reply:
done 646cfa3c0f570cb314a32521eedc099b8d321563